Compilation error on G++6 (flexible array member)

```c
In file included from /home/brain/rpmbuild/SPECS/libnitrokey/include/NitrokeyManager.h:6:0,
                 from /home/brain/rpmbuild/SPECS/libnitrokey/NitrokeyManager.cc:3:
/home/brain/rpmbuild/SPECS/libnitrokey/include/device_proto.h:137:27: error: flexible array member ‘nitrokey::proto::EmptyPayload::_data’ in an otherwise empty ‘struct nitrokey::proto::EmptyPayload’
             uint8_t _data[];
                           ^
/home/brain/rpmbuild/SPECS/libnitrokey/include/device_proto.h:136:16: note: in the definition of ‘struct nitrokey::proto::EmptyPayload’
         struct EmptyPayload {
                ^~~~~~~~~~~~
In file included from /home/brain/rpmbuild/SPECS/libnitrokey/include/NitrokeyManager.h:6:0,
                 from /home/brain/rpmbuild/SPECS/libnitrokey/NK_C_API.h:6,
                 from /home/brain/rpmbuild/SPECS/libnitrokey/NK_C_API.cc:2:
/home/brain/rpmbuild/SPECS/libnitrokey/include/device_proto.h:137:27: error: flexible array member ‘nitrokey::proto::EmptyPayload::_data’ in an otherwise empty ‘struct nitrokey::proto::EmptyPayload’
             uint8_t _data[];
                           ^
/home/brain/rpmbuild/SPECS/libnitrokey/include/device_proto.h:136:16: note: in the definition of ‘struct nitrokey::proto::EmptyPayload’
         struct EmptyPayload {
                ^~~~~~~~~~~~
In file included from /home/brain/rpmbuild/SPECS/libnitrokey/unittest/../include/NitrokeyManager.h:6:0,
                 from /home/brain/rpmbuild/SPECS/libnitrokey/unittest/../NK_C_API.h:6,
                 from /home/brain/rpmbuild/SPECS/libnitrokey/unittest/test_C_API.cpp:8:
/home/brain/rpmbuild/SPECS/libnitrokey/unittest/../include/device_proto.h:137:27: error: flexible array member ‘nitrokey::proto::EmptyPayload::_data’ in an otherwise empty ‘struct nitrokey::proto::EmptyPayload’
             uint8_t _data[];
                           ^
/home/brain/rpmbuild/SPECS/libnitrokey/unittest/../include/device_proto.h:136:16: note: in the definition of ‘struct nitrokey::proto::EmptyPayload’
         struct EmptyPayload {
                ^~~~~~~~~~~~
```

--- a/include/device_proto.h
+++ b/include/device_proto.h
@@ -134,8 +134,6 @@ namespace nitrokey {
         } __packed;
 
         struct EmptyPayload {
-            uint8_t _data[];
-
             bool isValid() const { return true; }
 
             std::string dissect() const { return std::string("Empty Payload."); }

